src/app/containers/redirect/redirect.component.spec.ts

```diff
@@ -1,8 +1,8 @@
-import { TestBed, async } from '@angular/core/testing';
-import { combineReducers, StoreModule, Store } from '@ngrx/store';
-import { reducers} from 'src/app/store';
 import { CUSTOM_ELEMENTS_SCHEMA } from '@angular/core';
+import { async, TestBed } from '@angular/core/testing';
+import { combineReducers, Store, StoreModule } from '@ngrx/store';
 import { cold } from 'jasmine-marbles';
+import { reducers} from 'src/app/store';
```

Comment:
@SivaprakasamSivaprakasam - change this import to use a relative path and correct spacing to `{ reducers }`